pkg/lvmutils/utils.go

```diff
@@ -73,8 +72,8 @@ func GetVgAvailableSize(vgname string) (uint64, uint64, error) {
 	return extentSize * freeExtents, extentSize, nil
 }
 
-// GetPoolMetadataSize calculates the thin pool metadata size based on the given thin pool size
-func GetPoolMetadataSize(poolsize uint64) uint64 {
+// GetTpMetadataSize calculates the thin pool metadata size based on the given thin pool size
+func GetTpMetadataSize(poolsize uint64) uint64 {
```

Comment:
we should have a minimum value for metadata size, for example, if a user creates with a pool size of 100MB, then metadata will be 0.5MB. We had a recent issue in a user setup where metadata became full because of small size

Reply:
I will look into that. But this is calculated based on given pool size. 

Refer https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.3/html-single/administration_guide/#Brick_Configuration 

Minimum metadata size required is 0.5% and Max upto 16GB

Reply:
please provide more details about the issue. This function is moved from `plugins/deviceutils` to `pkg/lvmutils`